PWGCF/TwoParticleCorrelations/TableProducer/identifiedBfFilter.h

```diff
@@ -26,6 +26,7 @@
 #include "Common/Core/TrackSelection.h"
 #include "Common/Core/TrackSelectionDefaults.h"
 #include "PWGCF/Core/AnalysisConfigurableCuts.h"
+#include "MathUtils/Utils.h"
 #include <TDatabasePDG.h>
```

Comment:
Using `TDatabasePDG.h` is highly inefficient
Please check https://indico.cern.ch/event/1482467/contributions/6246612/attachments/2973662/5235085/2024-11-25_O2-linter.pdf
and start moving to ` O2DatabasePDG service` in the iteration